app-android/src/main/java/com/ustadmobile/port/android/view/LanguageEditFragment.kt

```diff
@@ -72,4 +86,68 @@ class LanguageEditFragment: UstadEditFragment<Language>(), LanguageEditView {
             field = value
             mBinding?.fieldsEnabled = value
         }
+}
+
+@Composable
+fun LanguageEditScreen(
+    uiState: LanguageEditUiState,
+    onLanguageChanged: (Language?) -> Unit = {}
+){
+    Column (
+        modifier = Modifier
+            .padding(8.dp)
+            .verticalScroll(rememberScrollState()),
+    horizontalAlignment = Alignment.Start
+    ){
+
+        UstadTextEditField(
+            value = stringResource(id = R.string.name),
```

Comment:
Value for all fields should come from the entity on the UI state class, should not use the string constant.

e.g. should be:

value - uiState.language.name
label = stringResource(R.string.name)